backend/db/migrations/20191108194500-add-display-name-to-users.js

```diff
@@ -1,14 +0,0 @@
-export function up (queryInterface, Sequelize) {
```

Comment:
This file was breaking my migration. I believe this if from PR https://github.com/johnbyrne727rit/gallery-gateway/pull/4 (specifically here: https://github.com/johnbyrne727rit/gallery-gateway/blob/5344d0a37eab8a5a123e668c651e9bee96866466/backend/db/migrations/20191108194500-add-display-name-to-users.js#L1).

@moiseslorap Should this not be deleted? The column it's creating gets created in the `users` table migration already.